api/src/models/ImageFile.ts

```diff
@@ -71,21 +71,50 @@ class ImageFile {
     }
 
     ocrDerivative() {
-
+        let fs = require("fs");
+        let png = this.derivativePath('ocr/pngs', 'png');
+        let { exec } = require("child_process");
+        if (fs.existsSync(png)) {
+            let tc_cmd = "#{config['textcleaner_path']} #{config['textcleaner_switches']} #{derivative('LARGE')} #{png}"
```

Comment:
It looks like you copied the Ruby-style string embedding directly from the Ruby; this won't work as-is in Javascript. You'll need to pull these settings from the configuration object, and put the string together through concatenation. Also note that `derivative('LARGE')` is a call to get the path to the large derivative file.